### PR TITLE
Modify log format

### PR DESCRIPTION
**Pull request by yangzexia**
#6 

change:
1. add mask before vector register
2. merge mask, npc, rpc in one line
3. reverse order of output of simt_stack_entry and now the relative log more likes a stack.

### CODE REVIEW

File: gpgpu-testcase/driver/test.cpp [hunk 1]
```diff
@@ -19,9 +22,11 @@ struct meta_data{  // 这个metadata是供驱动使用的，而不是给硬件
       }
 };
 int main(){
-    uint64_t num_warp=2;
+    // printf("BUFFER SIZE: %d", PRINT_BUFFER_SIZE);
```
Comment (ghost) [inline gpgpu-testcase/driver/test.cpp]: Delete comment lines?

File: riscv/decode.h [hunk 2]
```diff
@@ -135,9 +135,9 @@ class insn_t
   uint64_t v_vta() { return x(26, 1); }
   uint64_t v_vma() { return x(27, 1); }
   uint64_t v_mew() { return x(28, 1); }
-  uint64_t v_simm12() {return xs(20,12);}
-  uint64_t v_simm11() {return xs(20,11);} 
-  uint64_t v_s_simm11() { return x(7, 5) + (xs(25, 6) << 5); }
+  uint64_t v_simm12() {return xs(20,12);} //for inst vls12 series
```
Comment (ghost) [inline riscv/decode.h]: keep comment style as below
`// for inst vls12 series`
space is needed after `//`

File: riscv/processor.h [hunk 3]
```diff
@@ -615,9 +616,9 @@ class processor_t : public abstract_device_t
           std::cout //<< "[is_part] " << is_part << " "
                     << "[r_pc] " << std::hex << std::setw(8) << std::setfill('0') << static_cast<uint32_t>(r_pc) << " " 
                     << "[else_pc] " << std::hex << std::setw(8) << std::setfill('0') << static_cast<uint32_t>(else_pc) << " " 
-                    << "[else_mask] " << std::hex << std::setw(8) << std::setfill('0') << else_mask << " " 
+                    << "[else_mask] " << std::hex << std::setw(8) << std::setfill('0') << else_mask << " ";
                     //<< "[pair] " << pair
```
Comment (ghost) [inline riscv/processor.h]: I noticed there are some comment lines. please remove

**Review by ghost — COMMENTED** [review]
I think maybe later, the whole program's coding style need to be changed, that will be a new PR

File: riscv/insns/vlw_v.h [hunk 4]
```diff
@@ -1,6 +1,9 @@
 VI_GPU_LD_INDEX(e32,true,({
     reg_t baseAddr = index[i] + insn.v_simm11();
-    reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
+    // reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
```
Comment (ghost) [inline riscv/insns/vlw_v.h]: Remove

File: riscv/insns/vsw_v.h [hunk 5]
```diff
@@ -1,7 +1,19 @@
...
+    reg_t baseAddr = index[i] + insn.v_s_simm11();
+    // reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
+    reg_t baseBias = P.get_csr(CSR_PDS) + (P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) * ((baseAddr >> 2)<<2));
+
+    // std::cout << "insn:"<<insn.bits()<<";index="<<index[i]<<";imm="<<insn.v_s_simm11() \
+    // <<";baseaddr="<<baseAddr<<";basebias="<<baseBias<<";vreg_inx="<<vreg_inx<<";store_addr="<<(baseBias+((baseTid + vreg_inx)<<2))<<std::endl;
+
+    MMU.store_uint32(baseBias+((baseTid + vreg_inx)<<2),P.VU.elt<uint32_t>(2,vs2, vreg_inx));
+    
+    // reg_t baseBias = P.get_csr(CSR_PDS) + (P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) * (baseAddr >> 2));
```
Comment (ghost) [inline riscv/insns/vsw_v.h]: Remove

File: riscv/insns/vsw_v.h [hunk 6]
```diff
@@ -1,7 +1,19 @@
-VI_GPU_ST_INDEX(e32,true,({
-    reg_t baseAddr = index[i] + insn.v_simm11();
-    reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
-    MMU.store_uint32(baseBias+((baseTid + vreg_inx)<<2),P.VU.elt<uint32_t>(2,vs2, vreg_inx));}
+VI_GPU_ST_INDEX(e32,true,(
+    {
+    reg_t baseAddr = index[i] + insn.v_s_simm11();
+    // reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
+    reg_t baseBias = P.get_csr(CSR_PDS) + (P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) * ((baseAddr >> 2)<<2));
+
```
Comment (ghost) [inline riscv/insns/vsw_v.h]: Remove

File: riscv/insns/vsw_v.h [hunk 7]
```diff
@@ -1,7 +1,19 @@
-VI_GPU_ST_INDEX(e32,true,({
-    reg_t baseAddr = index[i] + insn.v_simm11();
-    reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
-    MMU.store_uint32(baseBias+((baseTid + vreg_inx)<<2),P.VU.elt<uint32_t>(2,vs2, vreg_inx));}
+VI_GPU_ST_INDEX(e32,true,(
+    {
+    reg_t baseAddr = index[i] + insn.v_s_simm11();
+    // reg_t baseBias = P.get_csr(CSR_PDS) + (baseAddr & ~3) * P.get_csr(CSR_NUMW) * P.get_csr(CSR_NUMT) + (baseAddr & 3);
```
Comment (ghost) [inline riscv/insns/vsw_v.h]: Remove

**Comment by ghost** [comment]
LGTM

**Comment by guaguabao123** [comment]
LGTM

**Review by guaguabao123 — APPROVED** [review]

**Pull request merged**